runtime/common/src/crowdloan.rs

```diff
@@ -486,7 +486,8 @@ decl_module! {
 
 			// Try killing the crowdloan child trie
 			match Self::crowdloan_kill(index) {
-				child::KillOutcome::AllRemoved => {
+				// TODO use this value for refund
```

Comment:
I assume you do this in your other pr?

Reply:
yes

Reply:
dont wanna make this PR more than a companion